transport/http/handler.go

```diff
@@ -50,6 +51,11 @@ type handler struct {
 func (h handler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	start := time.Now()
 
+	if req.URL != nil && strings.HasPrefix(req.URL.Path, "/debug/") {
```

Comment:
I believe we want it to be a separate port, but I suppose we can grow into that stance.